dipy/workflows/atlasing.py

```diff
@@ -0,0 +1,101 @@
+            the same space and no registration is performed.
+        out_dir : str, optional
+            Output directory. If None, the current working directory is used.
+        merge_out : boolean, optional
+            If True the resulting atlases of all bundles are combined into a
+            single file. Default is False.
+        save_temp : boolean, optional
+            If True the intermediate results of each tree level are saved in a
+            temp folder in trk and png formats. Default is False.
+        n_stream_min : int, optional
+            Bundles with less than ``n_stream_min`` streamlines wont be
+            processed. Default is 10.
+        n_stream_max : int, optional
+            Bundles with more than ``n_stream_max`` streamlines are cropped to
+            have that number and speed up the computation. Default is 5000.
+        n_point : int, optional
+            All streamlines are set to have ``n_point`` points. Default is 20.
+        distance : str, optional
+            Distance metric to be used to combine bundles. Default is 'mdf'.
+            The 'mdf_se' distance uses only start/end points of streamlines.
+        comb_method : str, optional
+            Method used to combine each bundle pair. Default is 'rlap'.
+        skip_pairs : boolean, optional
+            If true bundle combination steps are randomly skipped. This helps
+            to obtain a sharper result. Default is False.
+        """
+        logging.info("workflow running")
+
+        if merge_out:
+            _, _ = compute_atlas_bundle(in_dir=in_dir,
+                                        subjects=subjects,
+                                        group=group,
+                                        mid_path=mid_path,
+                                        bundle_names=bundle_names,
+                                        model_bundle_dir=model_bundle_dir,
+                                        out_dir=out_dir,
+                                        merge_out=merge_out,
+                                        save_temp=save_temp,
+                                        n_stream_min=n_stream_min,
+                                        n_stream_max=n_stream_max,
+                                        n_point=n_point,
+                                        distance=distance,
+                                        comb_method=comb_method,
+                                        skip_pairs=skip_pairs)
+        else:
+            _ = compute_atlas_bundle(in_dir=in_dir,
+                                     subjects=subjects,
+                                     group=group,
+                                     mid_path=mid_path,
+                                     bundle_names=bundle_names,
+                                     model_bundle_dir=model_bundle_dir,
+                                     out_dir=out_dir,
+                                     merge_out=merge_out,
```

Comment:
You can call it one time and just always have it return two outputs. See my comment above in `compute_atlas_bundle `function.